packages/source-contentful/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "version": "0.5.3",
-  "name": "@gridsome/source-contentful",
+  "name": "@helloiamlukas/source-contentful",
```

Comment:
```suggestion
  "name": "@gridsome/source-contentful",
```

Reply:
sorry, this never belonged here 😔  reverted it now